docs/extensions/extension_rest_api.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Direct Phylum API Requests
+category: 62c5cb137dbdad00536291a6
+hidden: false
+---
+
+## Phylum REST API
+
+Phylum provides a versioned REST API for retrieving all available data. This
+REST API can be used directly by extensions if there is no TypeScript API
+available.
+
+All endpoins are documented here:
+https://api.staging.phylum.io/api/v0/swagger/index.html
+
+## Extension API requests
+
+To make a Request to Phylum's REST API, you can use the built-in
+`PhylumApi.fetch` function, which takes care of authentication and finding the
+correct base URI. The following example retrieves projects owned by the user
+which do not belong to any group:
+
+```ts
+import { PhylumApi, ApiVersion } from "phylum";
+
+// Create a fetch request to the `/data/projects/overview` endpoint.
+const reply = await PhylumApi.fetch(
+    ApiVersion.V0,
+    '/data/projects/overview',
+);
+
+// Parse the reply as JSON.
+const projects = await reply.json();
+
+// Output all our projects.
+console.log(projects);
+```
+
+Allowed options match the `fetch` API and can be overwritten to send more
```

Comment:
We could consider adding a link to [fetch in the deno docs](https://deno.land/api?s=fetch) or something.

But there are many places where users can find these docs, so maybe that's overkill.